src/strings.js

```diff
@@ -11,7 +11,8 @@
  *   '',  'bb'  => 'bb'
  */
 function concatenateStrings(value1, value2) {
-	throw new Error("Not implemented");
+	const ans = value1 + value2;
```

Comment:
You can return `value1 + value2` directly.

Reply:
ok bhaiya